src/JuMP.jl

```diff
@@ -77,108 +77,104 @@ const MOIBIN = MOICON{MOI.SingleVariable,MOI.ZeroOne}
 
 @MOIU.model JuMPMOIModel (ZeroOne, Integer) (EqualTo, GreaterThan, LessThan, Interval) (Zeros, Nonnegatives, Nonpositives, SecondOrderCone, RotatedSecondOrderCone, GeometricMeanCone, PositiveSemidefiniteConeTriangle, PositiveSemidefiniteConeSquare, RootDetConeTriangle, RootDetConeSquare, LogDetConeTriangle, LogDetConeSquare) () (SingleVariable,) (ScalarAffineFunction,ScalarQuadraticFunction) (VectorOfVariables,) (VectorAffineFunction,)
 
-###############################################################################
-# Model
+################################################################################
+# Model.
+# TODO(IainNZ): Clarify the three keyword arguments. `mode` is explained,
```

Comment:
This is being refactored in https://github.com/JuliaOpt/JuMP.jl/pull/1384

Reply:
Should hold off merging this for now or quickly get it in and build from it?

Reply:
Could you revert the docstring for `Model` and the TODO right here that's going to be out of date? Everything else seems fine.